enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/context/InteractiveQuestionProvider.tsx

```diff
@@ -72,7 +76,7 @@ export const InteractiveQuestionProvider = ({
       await onBeforeSave?.(sdkQuestion, saveContext);
       await handleSaveQuestion(question);
       onSave?.(sdkQuestion, saveContext);
-      await loadQuestion();
+      await loadAndQueryQuestion();
```

Comment:
This is just a renaming to make it clear what this does. Previously it wasn't clear if we only load a question or also make a query, but now it's super duper clear.